### PR TITLE
[feat] Create emojis.md

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I have added a new entry to the Markdown section. In other words, this pull request closes issue #2474 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @KTom101! :) I'm ready for your review! 😄

File: .vscode/settings.json
Comment (SSwiniarski) [inline .vscode/settings.json:1]: Need to remove this file.

File: content/markdown/concepts/emojis/emojis.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+CatalogContent: 
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In Markdown, **emojis** are symbols that are conveyed via text and/or shortcodes.
+
+## Syntax
+
+When using Markdown, you can just copy and paste emojis straight from sources like [Emojipedia](https://emojipedia.org/):
```
Comment (SSwiniarski) [inline content/markdown/concepts/emojis/emojis.md:20]:
```suggestion
When using Markdown, emojis can be copied and pasted straight from sources like [Emojipedia](https://emojipedia.org/):
```
No first- or second-person pronouns.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@CBID2 I did a first review and only had a couple of comments.

**Comment by CBID2** [comment]
> @CBID2 I did a first review and only had a couple of comments.

Just committed your suggestions @SSwiniarski! :)

**Review by SSwiniarski — APPROVED** [review]
@CBID2 It looks good for a second review. (I deleted the extra file.)

**Comment by CBID2** [comment]
> @CBID2 It looks good for a second review. (I deleted the extra file.)

Thanks @SSwiniarski! :) The only issue I have is the "Push/ci format:verify" test. It keeps failing for some reason.

**Comment by SSwiniarski** [comment]
> > @CBID2 It looks good for a second review. (I deleted the extra file.)
> 
> Thanks @SSwiniarski! :) The only issue I have is the "Push/ci format:verify" test. It keeps failing for some reason.

@CBID2 I did an `npm run format` on the file and it fixed the issue.

**Comment by CBID2** [comment]
> > > @CBID2 It looks good for a second review. (I deleted the extra file.)
> 
> > 
> 
> > Thanks @SSwiniarski! :) The only issue I have is the "Push/ci format:verify" test. It keeps failing for some reason.
> 
> 
> 
> @CBID2 I did an `npm run format` on the file and it fixed the issue.

Thanks

File: content/markdown/concepts/emojis/emojis.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Syntax
+
+When using Markdown, emojis can be copied and pasted straight from sources like [Emojipedia](https://emojipedia.org/):
+
+```md
+Markdown for a specific text image of emoji
+```
+
+Another method that is used are shortcodes, emojis being described through words. These vary from application to application, so it's important to refer to their Markdown style guide to ensure that the emojis are described accordingly.
```
Comment (KTom101) [inline content/markdown/concepts/emojis/emojis.md:26]:
```suggestion
Another method used are shortcodes, emojis being described through words. These vary from application to application, so it's important to refer to their Markdown style guide to ensure that the emojis are described accordingly.
```

File: content/markdown/concepts/emojis/emojis.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+An emoji is copied from a source like [Emojipedia](https://emojipedia.org/) and used to signify the role being described in the heading below:
+
+```md
+### Content Writing Team 📝
+```
+
+## Example 2: Emoji Shortcodes
+
+In the paragraph below, the shortcode is conveyed in accordance to [GitHub's standards](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji):
```
Comment (KTom101) [inline content/markdown/concepts/emojis/emojis.md:42]:
```suggestion
In the paragraph below, the shortcode is conveyed in accordance with [GitHub's standards](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji):
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@CBID2 Just a few minor changes from me.

File: content/markdown/concepts/emojis/emojis.md (hunk 2 above)
Comment (CBID2) [reply content/markdown/concepts/emojis/emojis.md]: Done

**Review by KTom101 — APPROVED** [review]
@CBID2 Looks good 👍

**Comment by KTom101** [comment]
@CBID2 It's live here: https://www.codecademy.com/resources/docs/markdown/emojis

I'm noticing that perhaps we should have made the title say "Emojis" with a capital "E" and "s" at the end. Are you able to open a PR to change the title in the metadata?

**Comment by CBID2** [comment]
> @CBID2 It's live here: https://www.codecademy.com/resources/docs/markdown/emojis
> 
> 
> 
> I'm noticing that perhaps we should have made the title say "Emojis" with a capital "E" and "s" at the end. Are you able to open a PR to change the title in the metadata?

Sure I'll do it later

**Comment by KTom101** [comment]
> > @CBID2 It's live here: https://www.codecademy.com/resources/docs/markdown/emojis
> > I'm noticing that perhaps we should have made the title say "Emojis" with a capital "E" and "s" at the end. Are you able to open a PR to change the title in the metadata?
> 
> Sure I'll do it later

Great, thanks!

**Comment by CBID2** [comment]
> > > @CBID2 It's live here: https://www.codecademy.com/resources/docs/markdown/emojis
> 
> > > I'm noticing that perhaps we should have made the title say "Emojis" with a capital "E" and "s" at the end. Are you able to open a PR to change the title in the metadata?
> 
> > 
> 
> > Sure I'll do it later
> 
> 
> 
> Great, thanks!

Can you review this one?

**Comment by CBID2** [comment]
> > > @CBID2 It's live here: https://www.codecademy.com/resources/docs/markdown/emojis
> > > I'm noticing that perhaps we should have made the title say "Emojis" with a capital "E" and "s" at the end. Are you able to open a PR to change the title in the metadata?
> > 
> > 
> > Sure I'll do it later
> 
> Great, thanks!

Made the change @KTom101! :) It's #2587

**Pull request merged**